frontend/templates/notification_history.html

```diff
@@ -3,11 +3,11 @@
 
 {%  block content %}
   <div class="title">
-    <h1>Notification history for <a href="/view_study/{{ study.id }}/participant/{{ participant.id }}">{{ participant.patient_id }}</a></h1>
+    <h1>Notification history for <a href="/view_study/{{ study.id }}/participant/{{ participant.patient_id }}">{{ participant.patient_id }}</a></h1>
 
     <p><b>Page of results:</b></p>
     <ul class="pagination" style="margin-top:0px;">
-      {% set urlbase = "/view_study/" + study.id|string + "/participant/" + participant.id|string + "/notification_history?page=" %}
+      {% set urlbase = "/view_study/" + study.id|string + "/participant/" + participant.patient_id|string + "/notification_history?page=" %}
```

Comment:
:pick: The `|string` may not be necessary anymore.

Reply:
Good catch: fixed.